sample-test-jmx-windows.yaml

```diff
@@ -0,0 +1,19 @@
+version: v0.1
```

Comment:
Can we use same files for ubuntu and windows? We can parameterize the testName to avoid conflicts

Reply:
changed